src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -832,7 +845,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
   -->
   <Target Name="_GenerateProjectRestoreGraphPerFramework"
-    DependsOnTargets="_GetRestoreProjectStyle;CollectPackageReferences;CollectPackageDownloads;CollectFrameworkReferences"
+    DependsOnTargets="_GetRestoreProjectStyle;CollectPackageReferences;CollectPackageDownloads;CollectFrameworkReferences;CollectCentralPackageVersions"
```

Comment:
off topic, but would it be worthwhile  adding a comment that changes to this list need to be reflected in `NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs` as well?